app/presenters/two_factor_authentication/selection_presenter.rb

```diff
@@ -9,6 +9,10 @@ def initialize(configuration: nil, user: nil)
       @user = user
     end
 
+    def render_in(_view_context, &block)
+      yield block
+    end
```

Comment:
For more info, this is a (new-ish) Rails API that lets an arbitrary class be passed as the argument for `render`.

See: https://api.rubyonrails.org/v6.1.4/classes/ActionView/Helpers/RenderingHelper.html#method-i-render